tableau.yaml

```diff
@@ -0,0 +1,118 @@
+    params:
+      workbookId: "The ID of the workbook to retrieve"
+  - name: get-view-data
+    description: Retrieves data in comma separated value (CSV) format for the specified view in a Tableau workbook.
+    params:
+      viewId: "The ID of the view to get data from"
+  - name: get-view-image
+    description: Retrieves an image of the specified view in a Tableau workbook. The width and height in pixels can be provided.
+    params:
+      viewId: "The ID of the view to get image from"
+      height: "Height in pixels - default: 800"
+      width: "Width in pixels - default: 800"
+  - name: list-all-pulse-metric-definitions
+    description: Retrieves a list of all published Pulse Metric Definitions using the Tableau REST API.
+    params:
+      view: "The range of metrics to return (DEFINITION_VIEW_BASIC, DEFINITION_VIEW_FULL, DEFINITION_VIEW_DEFAULT) - default: DEFINITION_VIEW_BASIC"
+  - name: list-pulse-metrics-from-metric-definition-id
+    description: Retrieves a list of published Pulse Metrics from a Pulse Metric Definition using the Tableau REST API.
+    params:
+      pulseMetricDefinitionID: "The ID of the Pulse Metric Definition to list metrics for (36 character UUID)"
+  - name: list-pulse-metrics-from-metric-ids
+    description: Retrieves a list of published Pulse Metrics from a list of metric IDs using the Tableau REST API.
+    params:
+      metricIds: "Array of Pulse Metric IDs to list metrics for"
+  - name: list-pulse-metric-subscriptions
+    description: Retrieves a list of published Pulse Metric Subscriptions for the current user using the Tableau REST API.
+    params:
+      random_string: "Dummy parameter for no-parameter tools - default: None"
+  - name: generate-pulse-metric-value-insight-bundle
+    description: Generate an insight bundle for the current aggregated value for Pulse Metric using Tableau REST API. You need the full information of the Pulse Metric and Pulse Metric Definition to use this tool.
+    params:
+      bundleRequest: "The request object to generate a bundle for, containing bundle_request with version, options, and input"
+      bundleType: "The type of bundle to generate (ban, springboard, basic, detail) - default: ban"
+
+env:
+  - name: Tableau Server URL
+    key: SERVER
+    description: Your Tableau Cloud or Server URL (e.g., https://your-site.online.tableau.com)
+    required: true
+    sensitive: false
+  - name: Tableau Site Name
+    key: SITE_NAME
+    description: The name of the Tableau site to use. If not provided, the default site will be used.
+    required: false
+    sensitive: false
+  - name: Personal Access Token Name
+    key: PAT_NAME
+    description: The name of your Personal Access Token
+    required: true
+    sensitive: false
+  - name: Personal Access Token Value
+    key: PAT_VALUE
+    description: The secret value of your Personal Access Token
+    required: true
+    sensitive: true
+
+runtime: npx
+npxConfig:
+  package: "@tybalex/tableau-mcp@1.7.0" 
```

Comment:
Do we want to package this up as a container, like we do with some of our other MCP servers? It seems unusual for us to just reference a package under an employee's NPM account.

Reply:
We can move it to the Obot NPM account.